### PR TITLE
Removes library DB stuff for being broken and unsupported

### PR DESCRIPTION
**Pull request by Iamgoofball**
Desucake said to do this.

### CODE REVIEW

**Comment by TehZombehz** [comment]
You have an uncanny ability to make ragecode in an amazingly short amount of time.

**Comment by Archie700** [comment]
Does anyone even bother with library?

File: code/modules/library/lib_machines.dm [hunk 1]
```diff
@@ -20,105 +20,13 @@
...
-			SQLquery += "author LIKE '%[author]%' AND title LIKE '%[title]%' AND category='[category]'"
-		screenstate = 1
-
-	if(href_list["back"])
-		screenstate = 0
-
-	src.add_fingerprint(usr)
-	src.updateUsrDialog()
+	user << "The machine explodes!"
+	ex_act(1)
```
Comment (PKPenguin321) [inline code/modules/library/lib_machines.dm]: dont you hate it when you try to log onto the computer at your local library and it explodes

Comment (Iamgoofball) [inline code/modules/library/lib_machines.dm]: this is the accurate library experience

**Pull request closed without merging**